api/HTMLElement.json

```diff
@@ -1981,10 +1981,10 @@
               "version_added": "45"
             },
             "safari": {
-              "version_added": "10"
+              "version_added": false
```

Comment:
@vinyldarkscratch can you back this out? It's `nonce`, updated in https://github.com/mdn/browser-compat-data/pull/7303.